bentoml/_internal/bento/bento.py

```diff
@@ -340,81 +341,69 @@ def __init__(self, base_path: t.Union[PathType, "FS"]):
         super().__init__(base_path, Bento)
 
 
-@attr.define(frozen=True, on_setattr=None)
```

Comment:
Why did you remove this? This was for compatibility with some (probably older) `attrs` version, I think.

We should definitely add a comment as explanation here, though.

Reply:
define is a new 22.1 feature anyway right?

Reply:
Sure, but there was at least one version where not doing this did something we didn't want it to do. Don't remember which one, though...

Reply:
Seems like it's in from v 21.

Reply:
I mean don't we already do `@attr.frozen` anyway?

Reply:
@sauyon we have `attrs>=21.1.0` in our dependency now, should be ok to just use `@attr.frozen` here now?